<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_ArcTangentialOptions</class>
 <widget class="QWidget" name="QG_ArcTangentialOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>22</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tangential Arc Options</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lRadius">
     <property name="text">
      <string>Radius:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leRadius">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="sep1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>leRadius</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QG_ArcTangentialOptions</receiver>
   <slot>updateRadius(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
